### ISSUE TITLE
Could this pipeline be applied to phase the sex chromosome of an autotetraploid plant? 

### ISSUE THREAD

**Issue by Huyuxi08** [tangerzhang/sexPhase#1]
Hi,

I'm trying to assemble the sex chromosomes of a dioecious autotetraploid plant. I have tried to directly use ALLHiC to assemble the whole genome, but it is difficult to separate different haplotypes of homologous chromosomes. Could this pipeline be applied to phase the sex chromosome of an autotetraploid plant? 

Thanks in advance for any help!

**Comment by tangerzhang** [comment]
No. This pipeline was not designed to process the sex chromosomes of auto-tetraploid. You can contact me through email (zhangxingtan@caas.cn) or leave messages on Github issues if you are facing problems with using ALLHiC.
